BUCK

```diff
@@ -2,7 +2,7 @@
 genrule(
     name = '_bundle-javascript',
     srcs = glob( [ 'src/main/javascript/**/*' ] ),
-    cmd = 'browserify --debug src/main/javascript/home.js > $OUT',
+    cmd = 'nodejs ./../../../node_modules/.bin/browserify --debug src/main/javascript/home.js -o $OUT',
```

Comment:
Not a lot  you can do about that without working out what the project root directory looks like. Buck doesn't make that easy so this will do fine for now.

You could list node_modules/ in the srcs glob but that's an awful lot of symlinks for buck to create for little benefit.

Or you could just make "npm install -g browserify" a setup job in the readme...